import-export-cli/utils/structs.go

```diff
@@ -48,7 +48,9 @@ type EnvEndpoints struct {
 	ApiManagerEndpoint      string `yaml:"api_manager_endpoint"`
 	ApiImportExportEndpoint string `yaml:"api_import_export_endpoint"`
 	ApiListEndpoint         string `yaml:"api_list_endpoint"`
+	AppListEndpoint			string `yaml:"application_list_endpoint"`
```

Comment:
Can we format this code using the GoLand IDE? It would fix the indentation issues.